### PR TITLE
so --> ,so

### PR DESCRIPTION
**Pull request by lixin963**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ The binaries published with this release are built with Go1.17.8 to avoid [CVE-2
 
 ## 2.33.0 / 2022-01-29
 
-* [CHANGE] PromQL: Promote negative offset and `@` modifer to stable features. #10121
```
Comment (roidelapluie) [inline CHANGELOG.md:101]: this is valid

File: CHANGELOG.md [hunk 2]
```diff
@@ -335,7 +335,7 @@ This vulnerability has been reported by Aaron Devaney from MDSec.
 * [ENHANCEMENT] Templating: Enable parsing strings in `humanize` functions. #8682
 * [BUGFIX] UI: Provide errors instead of blank page on TSDB Status Page. #8654 #8659
 * [BUGFIX] TSDB: Do not panic when writing very large records to the WAL. #8790
-* [BUGFIX] TSDB: Avoid panic when mmaped memory is referenced after the file is closed. #8723
+* [BUGFIX] TSDB: Avoid panic when mapped memory is referenced after the file is closed. #8723
```
Comment (roidelapluie) [inline CHANGELOG.md:338]: mmaped is correct

File: CHANGELOG.md [hunk 3]
```diff
@@ -1468,7 +1468,7 @@ This is a breaking change to the Kubernetes service discovery.
 * [FEATURE] EC2 service discovery: Expose `ec2_instance_state`.
 * [ENHANCEMENT] Various performance improvements in local storage.
 * [BUGFIX] Zookeeper service discovery: Remove deleted nodes.
-* [BUGFIX] Zookeeper service discovery: Resync state after Zookeeper failure.
+* [BUGFIX] Zookeeper service discovery: Rsync state after Zookeeper failure.
```
Comment (roidelapluie) [inline CHANGELOG.md:1471]: Resync is correct

File: RELEASE.md [hunk 4]
```diff
@@ -86,7 +86,7 @@ Check the [dependencies GitHub label](https://github.com/prometheus/prometheus/l
 This bot currently does not manage `+incompatible` and `v0.0.0` in the version specifier for Go modules.
 
 Note that after a dependency update, you should look out for any weirdness that
-might have happened. Such weirdnesses include but are not limited to: flaky
+might have happened. Such weirdness include but are not limited to: flaky
```
Comment (roidelapluie) [inline RELEASE.md:89]: Should we say includes then?

File: consoles/node.html [hunk 5]
```diff
@@ -20,7 +20,7 @@ <h1>Node</h1>
 {{ range query "up{job='node'}" | sortByLabel "instance" }}
 <tr>
   <td><a href="node-overview.html?instance={{ .Labels.instance }}">{{ reReplaceAll "(.*?://)([^:/]+?)(:\\d+)?/.*" "$2" .Labels.instance }}</a></td>
-  <td{{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
+  <td> {{ if eq (. | value) 1.0 }}>Yes{{ else }} class="alert-danger">No{{ end }}</td>
```
Comment (roidelapluie) [inline consoles/node.html:23]: This is incorrect

File: RELEASE.md (hunk 4 above)
Comment (beorn7) [reply RELEASE.md:89]: Yes, either "weirdnesses include but are" or "weirdness includes but is". But TBH the existing wording appears fine to me. It's a bit playful, but I don't see anything wrong with that. If you want a more compact and technical wording, I would probably do the following:

"Note that after a dependency update, you should look out for any weirdness that might have happened, including but not limited to flaky tests, differences in resource usage, and panics."

**Pull request closed without merging**